libs/openFrameworks/app/ofAppRunner.h

```diff
@@ -13,40 +13,34 @@ class ofBaseApp;
 class ofBaseRenderer;
 class ofCoreEvents;
 
+using std::shared_ptr;
```

Comment:
I don't recommend `using` declaration on header file.
It will affect to all codes include `ofAppRunner.h`.

Reply:
thank you @2bbb I'll be changing this